CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Ansible Modules for HPE OneView Change Log
 
+## v5.6.0 (Unreleased)
+This release extends the planned support of the modules to OneView REST API version 800 (OneView v4.1), 1000 (OneView v4.2), 1200 (OneView v5.0) and 1600 (OneView v5.2).
+
+#### Major changes
+1. Extended support of planned modules to API800/1000/1200/1600.
+2. Modules implemented in this release requires hpOneView version 5.2.0.
+
+#### Modules supported in this release
+- oneview_certificates_server
```

Comment:
Description doesn't match with the list of resources supported in the PR
As suggested earlier, it's not at all a good way to open PR with changes in multiple resources.

Reply:
closing this PR. Will open new PR separately for cluster profiles with the incorporated comments.